api_specifications/case-api-0.1.0.yaml

```
@@ -1340,3 +1396,100 @@ components:
+    Profile:
+      type: object
+      properties:
+        tenantId:
+          type: string
+          description: "id of the tenant"
+        litigantDetails:
+          type: object
+          properties:
+            partyType:
+              type: string
+              enum: [ "complainant", "respondent" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)